docs_user/modules/openstack-edpm_adoption.adoc

```diff
@@ -127,7 +127,7 @@ do
   ssh \
     -i ~/install_yamls/out/edpm/ansibleee-ssh-key-id_rsa \
     root@"${computes[$name]}" \
-                "echo $uuid | sudo tee /var/lib/nova/compute_id && sudo chown 42436:42436 /var/lib/nova/compute_id && sudo chcon -t container_file_t /var/lib/nova/compute_id"
+      "grep -qF $uuid /var/lib/nova/compute_id || (echo $uuid | sudo tee /var/lib/nova/compute_id && sudo chown 42436:42436 /var/lib/nova/compute_id && sudo chcon -t container_file_t /var/lib/nova/compute_id)"
```

Comment:
I guess we can merge but this will go away with https://issues.redhat.com/browse/OSPRH-2302 anyway, right?

Reply:
It will be deleted altogether when 17.3 will be on the CDN as in 17.3 the uuid file is generated by nova already.